recipes/maxbin2/meta.yaml

```diff
@@ -0,0 +1,28 @@
+build:
+  number: 0
+
+package:
+  name: maxbin2
+  version: "2.2.1"
+source:
+  fn: MaxBin-2.2.1.tar.gz
+  url: http://downloads.jbei.org/data/microbial_communities/MaxBin/getfile.php?MaxBin-2.2.1.tar.gz
+  md5: 3748e8f65b5bf18f8bd341210724cc5c
+requirements:
+  build:
+    - gcc # [linux]
+    - llvm # [osx]
+  run:
+    - libgcc # [linux]
+    - perl
```

Comment:
I don't think my collaborators need perl, but I'll check. Hopefully we can drop this.